### PR TITLE
BAEL-3285 | Simultaneous Spring WebClient calls

### PR DESCRIPTION
**Pull request by macieg**

### CODE REVIEW

File: spring-5-reactive-client/src/main/java/com/baeldung/reactive/simultaneouswebclient/Client.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
+package com.baeldung.reactive.simultaneouswebclient;
```
Comment (JonCook) [inline spring-5-reactive-client/src/main/java/com/baeldung/reactive/simultaneouswebclient/Client.java]: Instead lets maybe use com.baeldung.reactive.webclient.simultaneous for the package.

File: spring-5-reactive-client/src/main/java/com/baeldung/reactive/simultaneouswebclient/Client.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+import reactor.core.publisher.Flux;
+import reactor.core.scheduler.Schedulers;
+
```
Comment (JonCook) [inline spring-5-reactive-client/src/main/java/com/baeldung/reactive/simultaneouswebclient/Client.java]: This class needs to be formatted with the eclipse formatter:
https://github.com/eugenp/tutorials/blob/master/eclipse/formatter.xml

File: spring-5-reactive-client/src/test/java/com/baeldung/reactive/simultaneouswebclient/ClientIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import com.github.tomakehurst.wiremock.WireMockServer;
+
+import java.util.Arrays;
```
Comment (JonCook) [inline spring-5-reactive-client/src/test/java/com/baeldung/reactive/simultaneouswebclient/ClientIntegrationTest.java]: Lets' remove this unused import please and again update the package as mentioned before.

File: spring-5-reactive-client/src/test/java/com/baeldung/reactive/simultaneouswebclient/ClientIntegrationTest.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class ClientIntegrationTest {
+
```
Comment (JonCook) [inline spring-5-reactive-client/src/test/java/com/baeldung/reactive/simultaneouswebclient/ClientIntegrationTest.java]: This class also needs formatting correctly using the eclipse formatter we have.
https://github.com/eugenp/tutorials/blob/master/eclipse/formatter.xml
Thanks.

File: spring-5-reactive-client/src/test/java/com/baeldung/reactive/simultaneouswebclient/ClientIntegrationTest.java (hunk 3 above)
Comment (macieg) [reply spring-5-reactive-client/src/test/java/com/baeldung/reactive/simultaneouswebclient/ClientIntegrationTest.java]: fixed

File: spring-5-reactive-client/src/test/java/com/baeldung/reactive/simultaneouswebclient/ClientIntegrationTest.java (hunk 4 above)
Comment (macieg) [reply spring-5-reactive-client/src/test/java/com/baeldung/reactive/simultaneouswebclient/ClientIntegrationTest.java]: fixed

File: spring-5-reactive-client/src/main/java/com/baeldung/reactive/simultaneouswebclient/Client.java (hunk 2 above)
Comment (macieg) [reply spring-5-reactive-client/src/main/java/com/baeldung/reactive/simultaneouswebclient/Client.java]: fixed

**Pull request merged**